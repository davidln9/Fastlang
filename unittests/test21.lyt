
int main() {

	int x = 5;

	if (x == 3) {
		println("x is 3");
	} else if (x == 2) {
		println("x is 2");
	} else {
		println("x is not 3 or 2");
	}
}
