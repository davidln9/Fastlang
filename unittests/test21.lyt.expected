x is not 3 or 2
